src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -176,41 +181,90 @@ default ThreadMember getSelfThreadMember()
     /**
      * Gets a {@link ThreadMember} of this thread by their {@link Member}.
      *
-     * Note that this operation relies on the ThreadMember cache for this ThreadChannel, and so is likely to return null.
+     * Note that this operation relies on the {@link #getThreadMembers() ThreadMember cache} for this ThreadChannel.
+     * As the cache is likely to be unpopulated, this method is likely to return null.
      *
-     * Use of retrieveThreadMember(Member) is preferred instead, once it is released.
+     * Use of {@link #retrieveThreadMember(Member)} is preferred instead, once it is released.
      *
      * @param member
      *        The member to get the {@link ThreadMember} for.
      *
      * @return The {@link ThreadMember} of this thread for the given member.
+     *
+     * @throws NullPointerException
+     *         If the given member is null.
```

Comment:
This should not throw a `NullPointerException`. If it does, that's a bug. `member` should be checked by `Checks.notNull` and the proper `IllegalArgumentException` should be thrown instead.